opm/grid/cpgpreprocess/preprocess.c

```diff
@@ -190,7 +191,7 @@ checkmemory(int nz, struct processed_grid *out, int **intersections)
         if (ok) { out->m = m; }
     }
 
-    if (ok && (n != out->n)) {
+    if (ok && (n != ((size_t)(out->n))) {
```

Comment:
Looks like there's a right parenthesis (`)`) missing here.

Reply:
I fixed it now, but had to do a double take, so the current build that you triggered might not build against the correct version.